src/main/java/uk/gov/digital/ho/proving/income/audit/AuditClient.java

```diff
@@ -51,15 +53,19 @@
                 RestTemplate restTemplate,
                 RequestData requestData,
                 @Value("${pttg.audit.endpoint}") String auditEndpoint,
```

Comment:
The arg list is getting long. Probably worth thinking about how to address this.

Reply:
Agreed - I'll do that now